src/js/components/AppListFilterComponent.jsx

```diff
@@ -22,9 +23,13 @@ var AppListFilterComponent = React.createClass({
       filterText: filterText,
       activated: filterText !== "" || this.state.focused
     });
-    this.props.onChange(filterText);
+    this.fireChangeEvent(filterText);
```

Comment:
Hmm, looks like we can't have the fat arrow syntax until the next Babel release (see babel/#619), so I'll use a regular function definition instead.